Shared/Extensions/UIColorExtensions.swift

```diff
@@ -36,4 +36,19 @@ extension UIColor {
         Scanner(string: colorString).scanHexInt32(&colorInt)
         self.init(rgb: (Int) (colorInt))
     }
+
+    public var hexString: String {
+        let colorRef = cgColor.components
+        let r = colorRef?[0] ?? 0
+        let g = colorRef?[1] ?? 0
+        let b = ((colorRef?.count ?? 0) > 2 ? colorRef?[2] : g) ?? 0
+        let a = cgColor.alpha
+
+        var color = String(format: "#%02lX%02lX%02lX", lroundf(Float(r * 255)), lroundf(Float(g * 255)), lroundf(Float(b * 255)))
+        if a < 1 {
+            color += String(format: "%02lX", lroundf(Float(a)))
+        }
+
+        return color
+    }
```

Comment:
This function is used when returning a webpage with the custom scheme, and we want to theme the background. It turns the UIColor into a hex string for html use.